%% bare_jrnl.tex
%% V1.4a
%% 2014/09/17
%% by Michael Shell
%% see http://www.michaelshell.org/
%% for current contact information.
%%
%% This is a skeleton file demonstrating the use of IEEEtran.cls
%% (requires IEEEtran.cls version 1.8a or later) with an IEEE
%% journal paper.
%%
%% Support sites:
%% http://www.michaelshell.org/tex/ieeetran/
%% http://www.ctan.org/tex-archive/macros/latex/contrib/IEEEtran/
%% and
%% http://www.ieee.org/

%%*************************************************************************
%% Legal Notice:
%% This code is offered as-is without any warranty either expressed or
%% implied; without even the implied warranty of MERCHANTABILITY or
%% FITNESS FOR A PARTICULAR PURPOSE! 
%% User assumes all risk.
%% In no event shall IEEE or any contributor to this code be liable for
%% any damages or losses, including, but not limited to, incidental,
%% consequential, or any other damages, resulting from the use or misuse
%% of any information contained here.
%%
%% All comments are the opinions of their respective authors and are not
%% necessarily endorsed by the IEEE.
%%
%% This work is distributed under the LaTeX Project Public License (LPPL)
%% ( http://www.latex-project.org/ ) version 1.3, and may be freely used,
%% distributed and modified. A copy of the LPPL, version 1.3, is included
%% in the base LaTeX documentation of all distributions of LaTeX released
%% 2003/12/01 or later.
%% Retain all contribution notices and credits.
%% ** Modified files should be clearly indicated as such, including  **
%% ** renaming them and changing author support contact information. **
%%
%% File list of work: IEEEtran.cls, IEEEtran_HOWTO.pdf, bare_adv.tex,
%%                    bare_conf.tex, bare_jrnl.tex, bare_conf_compsoc.tex,
%%                    bare_jrnl_compsoc.tex, bare_jrnl_transmag.tex
%%*************************************************************************


% *** Authors should verify (and, if needed, correct) their LaTeX system  ***
% *** with the testflow diagnostic prior to trusting their LaTeX platform ***
% *** with production work. IEEE's font choices and paper sizes can       ***
% *** trigger bugs that do not appear when using other class files.       ***                          ***
% The testflow support page is at:
% http://www.michaelshell.org/tex/testflow/



\documentclass[journal]{IEEEtran}
%
% If IEEEtran.cls has not been installed into the LaTeX system files,
% manually specify the path to it like:
% \documentclass[journal]{../sty/IEEEtran}





% Some very useful LaTeX packages include:
% (uncomment the ones you want to load)


% *** MISC UTILITY PACKAGES ***
%
%\usepackage{ifpdf}
% Heiko Oberdiek's ifpdf.sty is very useful if you need conditional
% compilation based on whether the output is pdf or dvi.
% usage:
% \ifpdf
%   % pdf code
% \else
%   % dvi code
% \fi
% The latest version of ifpdf.sty can be obtained from:
% http://www.ctan.org/tex-archive/macros/latex/contrib/oberdiek/
% Also, note that IEEEtran.cls V1.7 and later provides a builtin
% \ifCLASSINFOpdf conditional that works the same way.
% When switching from latex to pdflatex and vice-versa, the compiler may
% have to be run twice to clear warning/error messages.






% *** CITATION PACKAGES ***
%
%\usepackage{cite}
% cite.sty was written by Donald Arseneau
% V1.6 and later of IEEEtran pre-defines the format of the cite.sty package
% \cite{} output to follow that of IEEE. Loading the cite package will
% result in citation numbers being automatically sorted and properly
% "compressed/ranged". e.g., [1], [9], [2], [7], [5], [6] without using
% cite.sty will become [1], [2], [5]--[7], [9] using cite.sty. cite.sty's
% \cite will automatically add leading space, if needed. Use cite.sty's
% noadjust option (cite.sty V3.8 and later) if you want to turn this off
% such as if a citation ever needs to be enclosed in parenthesis.
% cite.sty is already installed on most LaTeX systems. Be sure and use
% version 5.0 (2009-03-20) and later if using hyperref.sty.
% The latest version can be obtained at:
% http://www.ctan.org/tex-archive/macros/latex/contrib/cite/
% The documentation is contained in the cite.sty file itself.

\usepackage{hyperref}

\usepackage{color}


% *** GRAPHICS RELATED PACKAGES ***
%
\ifCLASSINFOpdf
   \usepackage[pdftex]{graphicx}
  % declare the path(s) where your graphic files are
  % \graphicspath{{../pdf/}{../jpeg/}}
  % and their extensions so you won't have to specify these with
  % every instance of \includegraphics
  % \DeclareGraphicsExtensions{.pdf,.jpeg,.png}
\else
  % or other class option (dvipsone, dvipdf, if not using dvips). graphicx
  % will default to the driver specified in the system graphics.cfg if no
  % driver is specified.
   \usepackage[dvips]{graphicx}
  % declare the path(s) where your graphic files are
  % \graphicspath{{../eps/}}
  % and their extensions so you won't have to specify these with
  % every instance of \includegraphics
  % \DeclareGraphicsExtensions{.eps}
\fi
% graphicx was written by David Carlisle and Sebastian Rahtz. It is
% required if you want graphics, photos, etc. graphicx.sty is already
% installed on most LaTeX systems. The latest version and documentation
% can be obtained at: 
% http://www.ctan.org/tex-archive/macros/latex/required/graphics/
% Another good source of documentation is "Using Imported Graphics in
% LaTeX2e" by Keith Reckdahl which can be found at:
% http://www.ctan.org/tex-archive/info/epslatex/
%
% latex, and pdflatex in dvi mode, support graphics in encapsulated
% postscript (.eps) format. pdflatex in pdf mode supports graphics
% in .pdf, .jpeg, .png and .mps (metapost) formats. Users should ensure
% that all non-photo figures use a vector format (.eps, .pdf, .mps) and
% not a bitmapped formats (.jpeg, .png). IEEE frowns on bitmapped formats
% which can result in "jaggedy"/blurry rendering of lines and letters as
% well as large increases in file sizes.
%
% You can find documentation about the pdfTeX application at:
% http://www.tug.org/applications/pdftex





% *** MATH PACKAGES ***
%
%\usepackage[cmex10]{amsmath}
% A popular package from the American Mathematical Society that provides
% many useful and powerful commands for dealing with mathematics. If using
% it, be sure to load this package with the cmex10 option to ensure that
% only type 1 fonts will utilized at all point sizes. Without this option,
% it is possible that some math symbols, particularly those within
% footnotes, will be rendered in bitmap form which will result in a
% document that can not be IEEE Xplore compliant!
%
% Also, note that the amsmath package sets \interdisplaylinepenalty to 10000
% thus preventing page breaks from occurring within multiline equations. Use:
%\interdisplaylinepenalty=2500
% after loading amsmath to restore such page breaks as IEEEtran.cls normally
% does. amsmath.sty is already installed on most LaTeX systems. The latest
% version and documentation can be obtained at:
% http://www.ctan.org/tex-archive/macros/latex/required/amslatex/math/





% *** SPECIALIZED LIST PACKAGES ***
%
%\usepackage{algorithmic}
% algorithmic.sty was written by Peter Williams and Rogerio Brito.
% This package provides an algorithmic environment fo describing algorithms.
% You can use the algorithmic environment in-text or within a figure
% environment to provide for a floating algorithm. Do NOT use the algorithm
% floating environment provided by algorithm.sty (by the same authors) or
% algorithm2e.sty (by Christophe Fiorio) as IEEE does not use dedicated
% algorithm float types and packages that provide these will not provide
% correct IEEE style captions. The latest version and documentation of
% algorithmic.sty can be obtained at:
% http://www.ctan.org/tex-archive/macros/latex/contrib/algorithms/
% There is also a support site at:
% http://algorithms.berlios.de/index.html
% Also of interest may be the (relatively newer and more customizable)
% algorithmicx.sty package by Szasz Janos:
% http://www.ctan.org/tex-archive/macros/latex/contrib/algorithmicx/




% *** ALIGNMENT PACKAGES ***
%
%\usepackage{array}
% Frank Mittelbach's and David Carlisle's array.sty patches and improves
% the standard LaTeX2e array and tabular environments to provide better
% appearance and additional user controls. As the default LaTeX2e table
% generation code is lacking to the point of almost being broken with
% respect to the quality of the end results, all users are strongly
% advised to use an enhanced (at the very least that provided by array.sty)
% set of table tools. array.sty is already installed on most systems. The
% latest version and documentation can be obtained at:
% http://www.ctan.org/tex-archive/macros/latex/required/tools/


% IEEEtran contains the IEEEeqnarray family of commands that can be used to
% generate multiline equations as well as matrices, tables, etc., of high
% quality.




% *** SUBFIGURE PACKAGES ***
%\ifCLASSOPTIONcompsoc
%  \usepackage[caption=false,font=normalsize,labelfont=sf,textfont=sf]{subfig}
%\else
%  \usepackage[caption=false,font=footnotesize]{subfig}
%\fi
% subfig.sty, written by Steven Douglas Cochran, is the modern replacement
% for subfigure.sty, the latter of which is no longer maintained and is
% incompatible with some LaTeX packages including fixltx2e. However,
% subfig.sty requires and automatically loads Axel Sommerfeldt's caption.sty
% which will override IEEEtran.cls' handling of captions and this will result
% in non-IEEE style figure/table captions. To prevent this problem, be sure
% and invoke subfig.sty's "caption=false" package option (available since
% subfig.sty version 1.3, 2005/06/28) as this is will preserve IEEEtran.cls
% handling of captions.
% Note that the Computer Society format requires a larger sans serif font
% than the serif footnote size font used in traditional IEEE formatting
% and thus the need to invoke different subfig.sty package options depending
% on whether compsoc mode has been enabled.
%
% The latest version and documentation of subfig.sty can be obtained at:
% http://www.ctan.org/tex-archive/macros/latex/contrib/subfig/




% *** FLOAT PACKAGES ***
%
%\usepackage{fixltx2e}
% fixltx2e, the successor to the earlier fix2col.sty, was written by
% Frank Mittelbach and David Carlisle. This package corrects a few problems
% in the LaTeX2e kernel, the most notable of which is that in current
% LaTeX2e releases, the ordering of single and double column floats is not
% guaranteed to be preserved. Thus, an unpatched LaTeX2e can allow a
% single column figure to be placed prior to an earlier double column
% figure. The latest version and documentation can be found at:
% http://www.ctan.org/tex-archive/macros/latex/base/


%\usepackage{stfloats}
% stfloats.sty was written by Sigitas Tolusis. This package gives LaTeX2e
% the ability to do double column floats at the bottom of the page as well
% as the top. (e.g., "\begin{figure*}[!b]" is not normally possible in
% LaTeX2e). It also provides a command:
%\fnbelowfloat
% to enable the placement of footnotes below bottom floats (the standard
% LaTeX2e kernel puts them above bottom floats). This is an invasive package
% which rewrites many portions of the LaTeX2e float routines. It may not work
% with other packages that modify the LaTeX2e float routines. The latest
% version and documentation can be obtained at:
% http://www.ctan.org/tex-archive/macros/latex/contrib/sttools/
% Do not use the stfloats baselinefloat ability as IEEE does not allow
% \baselineskip to stretch. Authors submitting work to the IEEE should note
% that IEEE rarely uses double column equations and that authors should try
% to avoid such use. Do not be tempted to use the cuted.sty or midfloat.sty
% packages (also by Sigitas Tolusis) as IEEE does not format its papers in
% such ways.
% Do not attempt to use stfloats with fixltx2e as they are incompatible.
% Instead, use Morten Hogholm'a dblfloatfix which combines the features
% of both fixltx2e and stfloats:
%
% \usepackage{dblfloatfix}
% The latest version can be found at:
% http://www.ctan.org/tex-archive/macros/latex/contrib/dblfloatfix/




%\ifCLASSOPTIONcaptionsoff
%  \usepackage[nomarkers]{endfloat}
% \let\MYoriglatexcaption\caption
% \renewcommand{\caption}[2][\relax]{\MYoriglatexcaption[#2]{#2}}
%\fi
% endfloat.sty was written by James Darrell McCauley, Jeff Goldberg and 
% Axel Sommerfeldt. This package may be useful when used in conjunction with 
% IEEEtran.cls'  captionsoff option. Some IEEE journals/societies require that
% submissions have lists of figures/tables at the end of the paper and that
% figures/tables without any captions are placed on a page by themselves at
% the end of the document. If needed, the draftcls IEEEtran class option or
% \CLASSINPUTbaselinestretch interface can be used to increase the line
% spacing as well. Be sure and use the nomarkers option of endfloat to
% prevent endfloat from "marking" where the figures would have been placed
% in the text. The two hack lines of code above are a slight modification of
% that suggested by in the endfloat docs (section 8.4.1) to ensure that
% the full captions always appear in the list of figures/tables - even if
% the user used the short optional argument of \caption[]{}.
% IEEE papers do not typically make use of \caption[]'s optional argument,
% so this should not be an issue. A similar trick can be used to disable
% captions of packages such as subfig.sty that lack options to turn off
% the subcaptions:
% For subfig.sty:
% \let\MYorigsubfloat\subfloat
% \renewcommand{\subfloat}[2][\relax]{\MYorigsubfloat[]{#2}}
% However, the above trick will not work if both optional arguments of
% the \subfloat command are used. Furthermore, there needs to be a
% description of each subfigure *somewhere* and endfloat does not add
% subfigure captions to its list of figures. Thus, the best approach is to
% avoid the use of subfigure captions (many IEEE journals avoid them anyway)
% and instead reference/explain all the subfigures within the main caption.
% The latest version of endfloat.sty and its documentation can obtained at:
% http://www.ctan.org/tex-archive/macros/latex/contrib/endfloat/
%
% The IEEEtran \ifCLASSOPTIONcaptionsoff conditional can also be used
% later in the document, say, to conditionally put the References on a 
% page by themselves.




% *** PDF, URL AND HYPERLINK PACKAGES ***
%
%\usepackage{url}
% url.sty was written by Donald Arseneau. It provides better support for
% handling and breaking URLs. url.sty is already installed on most LaTeX
% systems. The latest version and documentation can be obtained at:
% http://www.ctan.org/tex-archive/macros/latex/contrib/url/
% Basically, \url{my_url_here}.




% *** Do not adjust lengths that control margins, column widths, etc. ***
% *** Do not use packages that alter fonts (such as pslatex).         ***
% There should be no need to do such things with IEEEtran.cls V1.6 and later.
% (Unless specifically asked to do so by the journal or conference you plan
% to submit to, of course. )


% correct bad hyphenation here
\hyphenation{op-tical net-works semi-conduc-tor}

\def\SA#1{[{\color{red}SA: \it #1}]}
\def\JS#1{[{\color{magenta}JS: \it #1}]}

\usepackage{hyperref}
\hypersetup{
    colorlinks = true
}
\begin{document}
%
% paper title
% Titles are generally capitalized except for words such as a, an, and, as,
% at, but, by, for, in, nor, of, on, or, the, to and up, which are usually
% not capitalized unless they are the first or last word of the title.
% Linebreaks \\ can be used within to get better formatting as desired.
% Do not put math or special symbols in the title.
\title{CSE8803 Projects: Big Data Analytics for Healthcare }
%
%
% author names and IEEE memberships
% note positions of commas and nonbreaking spaces ( ~ ) LaTeX will not break
% a structure at a ~ so this keeps an author's name from being broken across
% two lines.
% use \thanks{} to gain access to the first footnote area
% a separate \thanks must be used for each paragraph as LaTeX2e's \thanks
% was not built to handle multiple paragraphs
%

\author{Jimeng Sun
\thanks{J. Sun is with School of Computational Science and Engineering Georgia Institute of Technology, Atlanta,
GA, 30332 USA e-mail: (see http://www.sunlab.org).}}


% The paper headers
\markboth{Workshop of CSE8803 Big Data Analytics for Healthcare, Fall 2016}%
{Shell \MakeLowercase{\textit{et al.}}: Bare Demo of IEEEtran.cls for Journals}

\maketitle

% As a general rule, do not put math, special symbols or citations
% in the abstract or keywords.
\begin{abstract}
CSE8803 Big Data Analytics for Healthcare is a graduate level course focusing on practical big data technology for health analytic applications. One big part of this course is to conduct an individual project that addresses a real-world data science problem in healthcare. The project should provide an end-to-end coverage of data science activities in addressing a real healthcare problem. The project should utilize big data systems such as Hadoop and Spark, machine learning algorithms that are covered in this class and real-world health related data. I hope that the best projects (with some additional effort) can  lead to publications at the best medical informatics venues such as \href{https://jamia.oxfordjournals.org/}{Journal of the American Medical Informatics Association (JAMIA)}, \href{http://www.journals.elsevier.com/journal-of-biomedical-informatics/}{Journal of Biomedical Informatics (JBI)}, \href{http://www.jmir.org/}{Journal of Medical Internet Research (JMIR)}, \href{http://www.journals.elsevier.com/artificial-intelligence-in-medicine/}{Artificial Intelligence in Medicine}, \href{http://jbhi.embs.org/}{IEEE Journal of Biomedical and Health Informatics (JBHI)}. 

This document provides the project guideline such as expectation, timeline, deliverables. We also introduce recommended project topics for selection but you are welcome to propose your own project as long as they are related to big data technology covered in this course and addressing healthcare problems. 
\end{abstract}

% Note that keywords are not normally used for peerreview papers.
\begin{IEEEkeywords}
Big data, Health analytics, Data mining, Machine learning
\end{IEEEkeywords}
% For peer review papers, you can put extra information on the cover
% page as needed:
% \ifCLASSOPTIONpeerreview
% \begin{center} \bfseries EDICS Category: 3-BBND \end{center}
% \fi
%
% For peerreview papers, this IEEEtran command inserts a page break and
% creates the second title. It will be ignored for other modes.
\IEEEpeerreviewmaketitle



\section{Introduction}
\IEEEPARstart{B}{ig} data and healthcare applications interact closely nowadays thanks to the advancement in electronic data capturing technology such as electronic health records, on-body sensors and genome sequencing. This course is about learning practical skills on big data systems, scalable machine learning algorithms and their applications to healthcare. Through (painful) homework exercises, all the students should have by now learned big data systems and acquired sufficient knowledge about healthcare data. We believe you are ready to take on the next level of challenges as a data scientist in healthcare. That is, you are going to propose, execute and report an awesome data science project. The final results of this project includes \textbf{ 1) a publishable report and 2) a convincing presentation, and 3) reusable software and sufficient documentation from your project.} 

Next we will cover the project life cycle, timeline, deliverables, grading scheme and project topics. 

\section{Project Life Cycle}
As a data scientist working on a real-world project, you have to be able to conduct all aspects of the big data project independently in a timely manner. In particular, here are some tasks that a data scientist will have to conduct in a big data project: project initiation, project execution and project report. 


\subsection{Project initiation}
As a data scientist, projects are not always there for you to work on. You have to create them and convince your boss (e.g., your CEO) to fund that. 
Before your project is officially launched, you have to conduct many steps to make that happen. 
Here are the checklist of things that you should do during the project initiation. 

\begin{enumerate}
\item Identify and motivate the problems that you want to address in your project.
\item Conduct literature search to understand the state of arts and the gap for solving the problem.
\item Formulate the data science problem in details (e.g.,  classification vs. predictive modeling vs. clustering problem). 
\item Identify clearly the success metric that you would like to use (e.g., AUC, accuracy, recall, speedup in running time). 
\item Setup the analytic infrastructure for your project (including both hardware and software environment, e.g., AWS or local clusters with Spark, python and all necessary packages).
\item Discover the key data that will be used in your project and make sure an efficient path for obtaining the dataset. This is a crucial step and can be quite time-consuming, so do it on the first day and never stops until the project completion.
\item Generate initial statistics over the raw data to make sure the data quality is good enough and the key assumption about the data are met. 
\item Identify the high-level technical approaches for the project (e.g., what algorithms to use or pipelines to use). 
\item Prepare a timeline and milestones of deliverables for the entire project.
\end{enumerate}

All the above steps in project initiation should be demonstrated in your proposal.
\subsection{Project execution }
Once your project is approved, you should quickly work on getting results and iterate with your sponsors on the progress. Iteration is the key. The first iteration should be fast and positive otherwise you are at risk losing momentum from the sponsors/project owners (e.g., your boss, clinical experts, your partners from another organization). This successful execution will lead to long-term sustainability of your team and will greatly improve your reputation in the organization, so please focus on that. 
\begin{enumerate}
\item Gather data that will be used in your project if you haven't already. 
\item Design the study (e.g., define cohort, target and features;  carefully split data into training, validation to avoid overfitting)
\item Clean and process the data. 
\item Develop and implement the modeling pipeline. 
\item Evaluate the model candidates on the performance metrics. 
\item Interpret the results from your model (e.g., show predictive features, compare to literature in terms of finding, present as cool visualization). 
\end{enumerate}

All the steps in project execution should be done by the paper draft due day and iterate at least another time by the final due day.
% needed in second column of first page if using \IEEEpubid
%\IEEEpubidadjcol
\subsection{Project report}
Finally, you are close to the end of the project. You need to summarize what you have done and learned throughout the project. This will be a comprehensive, concise and well-written report as the foundation for future projects. This can lead to publications and other external communication. You will probably need to give a presentation to your sponsors. So do the best you can in written report and presentation. Bad delivery at this step can overshadow all the great work your team have put in throughout the project, so do spend sufficient time to prepare a slick presentation and write a comprehensive report. 

\begin{itemize}
\item Your report should consists of the following sections.
\begin{enumerate}
\item Title and abstract
\item Introduction and background
\item Problem formulation
\item Approach and implementation
\item Experimental evaluation
\item Conclusion
\end{enumerate}
\item Prepare a presentation deck and deliver a convincing and informative presentation.
\item Clean up and package your code, and document the necessary steps for future usages by others.
\end{itemize}

Please use the above process to guide your own project for this semester and possibly your future data science career.

\section{Logistics}
Next we summarize the timeline and deliverables for your project in this semester. 

\subsection{Timeline}
\begin{center}
\begin{tabular}{ l |  l }
\hline
Due daterrrr & Task description \\
  \hline
  Oct 23 & Project proposal submission   \\
  Oct 30 & Peer review bidding\\
  Nov 20 & Project draft\\
  Nov 27 & Peer feedback for draft\\
  Dec 4  & Final paper and presentation\\
  Dec 11 & Peer feedback for final project\\
  \hline
\end{tabular}
\end{center}
Note that for peer review bidding, you need to login into easychair system for \href{https://easychair.org/conferences/?conf=cse8803bdhfall2016}{CSE8803BDH Workshop Fall 2016} and bid on 5 papers as willing to review. Eventually, 3 papers will be assigned to you for review. You will receive an email to be a PC member from easychair, and make sure you login and bid papers. 
\subsection{Deliverables}

\subsubsection{Project Proposal}
\begin{itemize}
\item 1-page write-up (word and latex templates will be provided)
\item Guide: 
\begin{itemize}
\item Explain about the problem/topic you choose and how to solve it
\item It is recommended to try to cover as many aspects as described in project initiation if it is possible.
\item Conduct literature search and cite at least 4 papers that are relevant to the project.
\end{itemize} 
\end{itemize}

\subsubsection{Project draft}
\begin{itemize}
\item up to 4-page write-up + 1 page reference
\item Guide
\begin{itemize}
\item Make sure your write-up cover all aspects described in project execution.
\item Conduct literature search and cite at least 8 papers or more that are relevant to the project.
\end{itemize}
\end{itemize}

\subsubsection{Peer feedback on others' paper draft}
You need to assess other's work based on the following criteria:
\begin{itemize}
\item Presentation quality
\item Importance of the problem
\item Comprehensive literature review
\item Feasible and meaningful approach
\item Clearly identified evaluation metric
\end{itemize}

\subsubsection{Final report}
\begin{itemize}
\item up to 5-page write-up + 1 page reference (the same template as project draft).
\item 5-min presentation (youtube or audio attached slides) + slides.
\item software implementation and documentation
\end{itemize}


\subsection{Grading scheme}
Here are the grading guideline for your project and peer participation.
\begin{itemize}
\item Project 45\%
\begin{itemize}
\item 5\% proposal
\item 10\% paper draft 
\item 12\% final presentation 
\item 18\% final paper (including Kaggle result)
\end{itemize}
\item Peer feedback (how you reviewed others papers) 6\%
\begin{itemize}
\item 2\% draft paper
\item 4\% final paper and presentation
\end{itemize}
\end{itemize}

\section{Project Topics}
We introduce several project topics for your consideration but you can also propose your own project outside this scope as long as your project uses big data tools (e.g., Hadoop and Spark) and is about healthcare applications. 

\subsection{Treatment Recommendation and Refractory Patient Management in Epilepsy}
Mentor: Sungtae An $\langle$stan84@gatech.edu$\rangle$

For these projects, you will develop predictive model for helping epilepsy patients. Two types of models should be considered 1) predicting refractory epilepsy patients and 2) predicting the treatment effectiveness using big data tools (Hadoop and Spark).
We suggest that you target one of the predictive models in your project. However, you can also propose your own project (upon approval) utilizing our large epilepsy dataset with big data analytics tools. 

\subsubsection{Predictive Modeling of Refractory Epilepsy Patients}
Drug-resistant epilepsy is a major clinical and societal problem for one in three epilepsy patients. We call this drug-resistant epilepsy population as refractory epilepsy patients. More specifically, we can define the refractory epilepsy population as epilepsy patients who failed with 3 distinct anti-epilepsy drugs (AED). In this project, you will work on building a predictive model to determine whether a patient will likely be a refractory epilepsy patient in advance, especially at their first AED failure time. Please refer to this \href{https://www.overleaf.com/read/qbxymxrpzqhg}{Description} for more details. This project includes Kaggle in Class competition among students who choose this topic.

\subsubsection{Treatment Recommendation for Epilepsy Patients}
In this project, you will develop a model to recommend optimal epileptic treatment regimens to patients. For this purpose of the project, you can use any kind of machine learning techniques. For example, you can build a predictive model for each regimen, which consists of a single AED or multiple AEDs, separately. Please refer to this \href{https://www.overleaf.com/read/ncmybvjkjcfs}{Description} for more detail information. We show an example approach in the description, but it is not restricted to apply any method you want to use.

\begin{itemize}
	\item \textbf{Resources}:  \href{https://www.overleaf.com/read/qbxymxrpzqhg}{Description for refractory epilepsy patient prediction}, \href{https://www.overleaf.com/read/ncmybvjkjcfs}{Description for epilepsy treatment recommendation}. Here are some related papers to get you started with the clinical challenges in epilepsy treatment~\cite{kwan_early_2000,schmidt_drug_2005,gilioli_focal_2012,voll_predicting_2015}. You are welcome to discuss the ideas with the mentor. 
   	\item \textbf{Dataset}: It will be provided through the secure environment.
    \item \textbf{Metrics}: AUC, sensitivity, specificity, scalability, etc.
    \item \textbf{Challenges}: Exact cohort construction, advanced feature engineering and processing big data for those steps.
\end{itemize}

\subsection{Septic shock prediction}
Mentor: Ruiming Lu $\langle$rlu39@gatech.edu$\rangle$

Sepsis is a leading cause of death in the United States, with mortality highest among patients who develop septic shock. Early aggressive treatment decreases morbidity and mortality. While general-purpose illness severity scoring systems are useful for predicting general deterioration or mortality, they typically cannot distinguish with high sensitivity and specificity which patients are at highest risk of developing specific acute condition.

Using supervised learning, a machine learning methodology, and the MIMIC (Multiparameter Intelligent Monitoring in Intensive Care)–-II Clinical Database (40), Henry et al.~\cite{henry_targeted_2015} trained a predictive model based on a targeted real-time early warning score (TREWScore) that identifies those patients at high risk of developing septic shock in the future. With a median lead time of over 24 hours, this scoring algorithm may allow clinicians enough time to intervene before the patients suffer the most damaging effects of sepsis.

The goal of the project is to repeat and improve the predictive model from \cite{henry_targeted_2015} using MIMIC-–III database~\cite{saeed_multiparameter_2011} with big data tools (Hadoop and Spark). The analytic process should follow the predictive modeling pipeline covered in the lecture. The detailed steps should be presented including prediction target, cohort construction, feature construction, feature selection, predictive model and performance evaluation. The key phases are contructing the features and building the predictve model.

The following are some guidelines for this project, but it is OK to deviate from the guidelines as long as you clearly state what you did and why it makes sense. For cohort construction, you can use ICD codes, criteria given in the paper, or other meaningful method. There is no date associated with the diagnostic cod. Therefore, you have to rely on looking at when the set of criteria given in the paper are satisfied to determine the time of onset of septic shock. Any other method are also welcome. Henry et al.~\cite{henry_targeted_2015} obtained With a median lead time of over 24 hours to predict, feel free to start off using that as the length of your prediction window and see how your model performs compared to what the paper obtained. The performance should be validated by cross-validation or a separate hold-off set. It is also expected that there are some scalability results reported, e.g., runtime vs no. of patients, runtime vs no. of cores, since you are using big data tools in this project. 

This project includes a Kaggle in Class competition among students who choose this topic.

\begin{itemize}
	\item \textbf{Resources}:  \href{http://stm.sciencemag.org.prx.library.gatech.edu/highwire/filestream/197913/field_highwire_adjunct_files/0/7-299ra122_SM.pdf}{Supplemental material for the paper}.
   	\item \textbf{Dataset}: \href{https://mimic.physionet.org/gettingstarted/dbsetup/}{MIMIC III}.
    \item \textbf{Metrics}: AUC, sensitivity, specificity, scalability, etc.
    \item \textbf{Challenges}: Feature construction and implementation using big data
\end{itemize}


\subsection{Mortality Prediction in ICU}
Mentor: Nisheeth Bandaru $\langle$nisheeth@gatech.edu$\rangle$

``Accurate knowledge of a patient’s disease state and trajectory
is critical in a clinical setting. Modern electronic healthcare
records contain an increasingly large amount of data,
and the ability to automatically identify the factors that
influence patient outcomes stand to greatly improve the efficiency and quality of care.

Ghassemi et al.~\cite{ghassemi_unfolding_2014} examined the use of latent variable models (viz. Latent Dirichlet Allocation) to decompose free-text hospital
notes into meaningful features, and the predictive power of
these features for patient mortality. This work considered three
prediction regimes: (1) baseline prediction, (2) dynamic (time-varying)
outcome prediction, and (3) retrospective outcome
prediction. In each, our prediction task differs from the
familiar time-varying situation whereby data accumulates;
since fewer patients have long ICU stays, as we move forward
in time fewer patients are available and the prediction
task becomes increasingly difficult.''

The goal of this project is to repeat and improve all or part of the study\cite{ghassemi_unfolding_2014} using the newer MIMIC-III data\cite{saeed_multiparameter_2011} implemented with big data tools (e.g., Hadoop and Spark). You must present detailed steps such as the prediction target, feature selection, feature construction, predictive model and performance evaluation.\\
You may inititally start with a small subset of data as you develop your model locally. However, after fine-tuning it, your final paper must be based on results from the entire data. You may also want to narrow your focus on implementing a subset of the models discussed in the paper. %While it is recommended that you at least have one time-varying model (.e.,g, as its more useful than retrospective models, you are welcome to implement a different model of your choice that does significant feature engineering.\\
The model must be evaluated by cross-validation and its performance on a hold-off test set. This project includes a Kaggle in Class competition among students who choose this topic.

\begin{itemize}
	\item \textbf{Resources}: \href{http://mghassem.mit.edu/wp-content/uploads/2013/02/ghassemi_naumann_kdd2014.pdf}{Paper},  \href{http://mghassem.mit.edu/wp-content/uploads/2013/02/Ghassemi_KDD2014_Presentation.pdf}{Presentation}, \href{http://videolectures.net/kdd2014_ghassemi_physiological_state/}{Video}.
   	\item \textbf{Dataset}: \href{https://mimic.physionet.org/gettingstarted/dbsetup/}{MIMIC III}.
    \item \textbf{Key Deliverable}: Models built with MIMIC-III data with similar or better performance.
    \item \textbf{Metrics}: AUC, sensitivity, specificity, scalability, etc.
    \item \textbf{Challenges}: Feature construction and implementation using big data tools.
\end{itemize}

% \subsection{Intelligent Interoperability for Health Data}
% Health data interoperability is a challenging but extremely important area in healthcare. Moving data from one data schema to another can be extremely painful and error-prone. 

% The project aims at developing an automatic schema mapping algorithms from one data health data format to another. In this case, we suggest you use \href{http://www.ohdsi.org/data-standardization/the-common-data-model/} {OMOP} and \href{https://www.hl7.org/fhir/json.html}{FHIR} as two reference health data standards. 

% Data: ExactData

\subsection{OHDSI Analytics with Big Data}
Mentor: Adrian Chang $\langle$adrian.chang@gatech.edu$\rangle$, Peter Schneider $\langle$peteryschneider@gatech.edu$\rangle$

The Observational Health Data Science and Informatics (\href{OHDSI}{http://www.ohdsi.org/analytic-tools}) is a community that focus on developing open source health care analytic tools. Unfortunately, most of those tools are built around traditional technologies are not designed to handle the large volume of data that modern healthcare creates. 

For this project, you can either develop a big data version of an existing OHSDI tool or create a new one based on the big data tools (Hadoop, Spark) that we have learned so far in this class. For example, if you were to choose ACHILLES to rewrite, part of your project would be to rewrite all of their SQL data processing in Spark. While developing the tool, you must also consider the data format (OMOP) and the way OHDSI tools store their data and see if either can be improved upon in addition to the actual data processing. By the end of the project, your tool should be able to be reused by other people and be polished enough that it could be published as an OHDSI tool. You are welcome to conduct your project leveraging the data such as MIMIC III and CMS synpuf.

If you choose this project, your project will primarily be evaluated on the following:

\begin{enumerate}
	\item 
    Reusability
    
    Does your tool produce consistent results that require little effort to setup and reproduce?
    \item
    Scalability
    
  	What is the limit of data that your tool can handle? Is it much better than an equivalent tool?
    \item
    Accuracy
    
    Does your tool produce comparable if not better results than the original tool or an equivalent tool?
    Did this occur because you chose a different algorithm to produce your results or because of something 
    else?
    
    \item 
    Ingenunity
    
    Is your tool taking a same approach to solve the same problem? Or did you invite something new?
\end{enumerate}

\begin{itemize}
	\item \textbf{Key Deliverable}: Reusable big data healthcare software transplanted from OHDSI.
   	\item \textbf{Input Data}: OMOP compliant database (e.g,. MIMIC III and CMS synpuf).
    \item \textbf{Metrics}: Scalability, accuracy, reusability.
    \item \textbf{Challenges}: Design and implementation of OHDSI tools and familiarity with the OMOP data format. 
\end{itemize}

\subsection{Other projects}
You are welcome to propose your own projects as long as 1) they are health analytic projects
and 2) they use big data tools covered in this class (Hadoop, Spark). Note that we will not provide much support on those projects. 

\section{Conclusion}
Best of luck on your project and data science rocks!




% use section* for acknowledgment
\section*{Acknowledgment}
Thanks all the TAs for their time and effort in creating the course material together. Thank all the students for their dedication and feedback.

\bibliography{cse8803project}
\bibliographystyle{abbrv}


\end{document}